### ISSUE TITLE
Context should implement From<serde_json::Value>

### ISSUE THREAD

**Issue by pchampin** [sunng87/handlebars-rust#477]
I have a `serde_json::Value` that I would like to use directly as the data of a template.
Using `render` is possible (since value implements `Serialize`) but my understanding is that the Value will be serialized, and deserialized again, which is a waste of CPU time...
I would like to convert my `Value` into a `Context` so I can use `render_with_context`, but that does not seem to be possible...

Am I missing something?

**Comment by sunng87** [comment]
You can use `Context::wraps` to convert `Value` into a reusable `Context`.

**Comment by pchampin** [comment]
But that will suffer from thé same problem as `render`: `wraps` expects any `Serialze` soi t will serialize and deserialize it again... That's precisely thé kind of useless roundtrip that I want to avoid...

**Comment by pchampin** [comment]
Not sure why you need to change `Context` with a `Cow`...
An impl of `From<serde_json::value::Value>` would be as simple as
```rust
impl From<serde_json::value::Value> for Context {
  fn from(data: serde_json::value::Value) -> Self {
    Context { data }
  }
}
```

**Comment by pchampin** [comment]
Maybe I'll explain my use-case more clearly.
My web application is retrieving data from a (local) REST API, which produces `application/json` responses. I parse these responses as a `serde_json::Value`, and I would like to pass that value *as is* (or cheaply wrapped in a `Context`)  to Handlebars, without the useless cost of a serialize-deserialize round-trip.

I do not intend to reuse the corresponding context in multiple calls to `render_with`.

**Comment by sunng87** [comment]
You are right. I missed the point. It's reasonable to have the conversion trait for Context.

**Comment by sunng87** [comment]
released in 4.1.6
